src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -55,8 +57,9 @@
     //private int autoHostile = 0;//Off
 
     //  Buildings
-    private BuildingTownHall townHall;
-    private Map<BlockPos, AbstractBuilding> buildings = new HashMap<>();
+    private BuildingTownHall                                townHall;
+    private                 Map<BlockPos, AbstractBuilding>         buildings                       = new HashMap<>();
+    final private                 Map<BlockPos, Field>            fields                          = new HashMap<>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)